core/templates/dev/head/pages/notifications_dashboard/NotificationsDashboard.js

```diff
@@ -20,8 +20,13 @@ oppia.controller('DashboardNotifications', [
     '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
     function($scope, $http, $rootScope, oppiaDatetimeFormatter) {
   $scope.navigateToItem = function(activityId, notificationType) {
```

Comment:
I don't think this function is being used anywhere now. If not, please remove it.